gramps/gen/lib/serialize.py

```diff
@@ -57,20 +71,20 @@ def __getattr__(self, key):
         if key.startswith("_"):
             raise AttributeError("can't use this API to access hidden attributes")
 
-        if key in self:
+        if "_object" in self:
```

Comment:
If performance is ever an issue you could make _object a direct attribute. This would avoid having to lookup _object each time in the dictionary.

Reply:
Probably a minor impact... but it is true that the data could already have an object in it, `{"_object": object}`. Changing to an attribute would require some refactoring.